contrib/initramfs/scripts/zfs

```diff
@@ -435,6 +439,7 @@ decrypt_fs()
 			# Prompt with ZFS tty, otherwise
 			else
 				# Setting "printk" temporarily to "7" will allow prompt even if kernel option "quiet"
```

Comment:
nit: Reword to something like: Temporarily setting "printk" to "7" allows the prompt to appear even when the "quiet" kernel option has been used.